### ISSUE TITLE
Ability to highlight and copy images

### ISSUE THREAD

**Issue by kele888** [GitMurf/roam-highlighter#36]
I think this would add immensely to the value of the extension. Often the images are an important part of the article/text.

**Comment by GitMurf** [comment]
Closed Issue #42 as a duplicate.

**Comment by kele888** [comment]
before copying the image for real, probably it would be easier at first to just add
![image title](image link on the web)

**Comment by GitMurf** [comment]
@kele888 yes I agree that’s the plan. In fact I don’t see it ever being an option to actually copy the image with the way my highlighter works so this will have to be the solution.

**Comment by wshuyi** [comment]
I want this feature too. :)

Thanks for the great roam-highlighter extension. It helped me a lot!

**Comment by jeremynevans** [comment]
+1 !
